Dockerfile_mbtci_template

```diff
@@ -248,6 +249,11 @@ RUN set -ex \
   && chown -R ${MTA_USER}:${MTA_USER} ${MTA_USER_HOME}
 ENV NPM_CONFIG_PREFIX ${MTA_USER_HOME}/.npm-global
 
+# Instll curl and ca-certificates
+RUN set -ex \
+  && apt-get update \
+  && apt-get install -y curl ca-certificates --no-install-recommends
```

Comment:
I doubt you need to install these packages: 
+ ca-certificates is already installed before; 
+ curl is not conform to SAP security policy for build environment. Any user relying on curl or wget must ask for an exception and build their own build env.